manifests/y/Yubico/YubikeyPersonalizationGUI/3.1.25/Yubico.YubikeyPersonalizationGUI.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Yubico.YubikeyPersonalizationGUI

```

```suggestion
PackageIdentifier: Yubico.YubiKeyPersonalizationTool
```
Edit filename and folder using github.dev